lib/Controller/FilesIntegrationController.php

```diff
@@ -112,6 +117,10 @@ public function __construct(
 	 * @throws OCSNotFoundException
 	 */
 	public function getRoomByFileId(string $fileId): DataResponse {
+		if ($this->config->getAppValue('spreed', 'conversations_files', '1') !== '1') {
+			throw new OCSException($this->l->t('File is not shared, or shared but not with the user'), Http::STATUS_UNAUTHORIZED);
```

Comment:
The exception message does not match the reason why it was thrown.

Reply:
yeah, it's the same for all cases so I just picked the same. I guess this is to not reveal if a file id exists and everything. So we could do a different one here, but not sure it's too useful